prow/scripts/cluster-integration/helpers/fluent-bit-stackdriver-logging.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# prepare stackdriver logging integration
+# - creates secret which holds ServiceAccount with LogWritter permission (for Stackdriver access)
+# - creates installation override for fluent-fit
+function prepare_stackdriver_logging() {
+    local filepath="${1}"
+    if [[ -z "${filepath}" ]]; then
+        echo "Filepath for installation override not given"
```

Comment:
You may consider using standard output method `shout` is already available for use as `library.sh` is already sourced.

This is true for all the `echo`s used.